packages/cli-repl/package.json

```diff
@@ -50,6 +50,7 @@
     "analytics-node": "^3.4.0-beta.1",
     "ansi-escape-sequences": "^5.1.2",
     "askpassword": "^1.2.3",
+    "aws4": "^1.11.0",
```

Comment:
This should be in service-provider-server if it's a peer dependency of the driver, which is also depended on there, right? (I think this currently would work by accident, because aws4 appears in this repository's top-level node_modules)

Reply:
Ah shoot... yeah it should go there then, absolutely! Interestingly it's not even a peer dependency of the driver, they only depend on `@types/aws4`.